group.go

```diff
@@ -51,3 +68,50 @@ func (s *GroupService) Get(name string) ([]GroupMember, *Response, error) {
 
 	return group.Members, resp, nil
 }
+
+// Add adds user to group
+//
+// JIRA API docs: https://docs.atlassian.com/jira/REST/cloud/#api/2/group-addUserToGroup
+func (s *GroupService) Add(groupname string, username string) (*Group, *Response, error) {
+	apiEndpoint := fmt.Sprintf("rest/api/2/group/user?groupname=%s", groupname)
+	var user struct {
+		Name string `json:"name"`
+	}
+	user.Name = username
+	req, err := s.client.NewRequest("POST", apiEndpoint, &user)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	responseGroup := new(Group)
+	resp, err := s.client.Do(req, responseGroup)
+	if err != nil {
+		jerr := NewJiraError(resp, err)
+		return nil, resp, jerr
+	}
+
+	return responseGroup, resp, nil
+}
+
+// Remove removes user from group
+//
+// JIRA API docs: https://docs.atlassian.com/jira/REST/cloud/#api/2/group-removeUserFromGroup
+func (s *GroupService) Remove(groupname string, username string) (*Response, error) {
+	apiEndpoint := fmt.Sprintf("rest/api/2/group/user?groupname=%s&username=%s", groupname, username)
+	// var user struct {
```

Comment:
Unless they're needed as documentation, I'd get rid of the comments.

Reply:
Oops, sorry. You're right.